bokeh/charts/data_source.py

```diff
@@ -761,7 +761,7 @@ def from_arrays(cls, arrays, column_names=None, **kwargs):
             arrays = list(chain.from_iterable(arrays))
             column_names = column_names or gen_column_names(len(arrays))
             cols = copy(column_names)
-            dims = kwargs.get('dims', None) or DEFAULT_DIMS
+            dims = kwargs.get('dims') or DEFAULT_DIMS
```

Comment:
As we are improving code quality, then this should be `dims = kwargs.get('dims', DEFAULT_DIMS)`.